docs/whatsnew/0.6.0.rst

```diff
@@ -0,0 +1,5 @@
+Plasmapy v0.6.0 (to be released mid-march)
```

Comment:
You probably don't need the title, towncrier will render one.

Reply:
Fair point!

Reply:
It does, however, set the title to `Plasmapy 0.5.1.dev87+g7438c409.d20210301 (2021-03-02)`, a bugfix release.

Reply:
If the next release that will be made off of your master branch is going to be v0.6.0 you should add a v6.0.0 tag on master so that setuptools_scm knows what version to render. As described here: https://packaging-guide.openastronomy.org/en/latest/releasing.html#releasing-from-branches